Simulation/RK4_Integration.m

```diff
@@ -5,10 +5,10 @@
 
     % Initialize with zeroed delta
     k0.P_dot = [0,0]; % Position Derivative
-    k0.V_dot = [0,0]; % Velocity Derivative
+    k0.V_dot = [0,0]; % Velocity Derivative thrust here 
```

Comment:
I don't think this was meant to be kept here? Or at least it doesn't read properly